box2d/gradle.properties

```diff
@@ -1 +1 @@
-jtranscVersion=0.6.3
+jtranscVersion=0.6.4-SNAPSHOT
```

Comment:
If you really need this, I'm going to accept this until I find a better way.
The problem with this is that if someone downloads the repo right now, they will find that it doesn't work. And will think that this is broken.

After checking if I have to fix something for the icons part in jtransc I will release (probably today 0.6.4)